service/prometheus/scrapeconfig_test.go

```diff
@@ -273,85 +335,6 @@ func Test_Prometheus_GetScrapeConfigs(t *testing.T) {
 				},
 			},
 		},
-
-		// Test that two services that specify the same cluster annotation create a scrape config together.
```

Comment:
This is the functionality we no longer need, as we only need the singular api / master service.